### PR TITLE
[ADD] Lunch: new configuration doc

### PR DESCRIPTION
**Pull request by larm-odoo**
New doc explaining how to set up the app

### CODE REVIEW

**Comment by larm-odoo** [comment]
Thanks for volunteering, @samueljlieber! Just an FYI, the failure in the check above is because there's no codeowner for the lunch app yet (since the app doesn't currently exist in dicumentation).

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:53]: This notice doesnt really apply to the translations form, since this is one of the instances where the "Save" button is always present. I'd suggest removing this or moving it somewhere more relevant :)
```suggestion
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:67]: Icon formatting
```suggestion
click the :guilabel:`≣ (four horizontal lines)` icon in the top-left corner.
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:74]: Sentence style heading, since this isnt a form walkthrough and no UI text stating "Vendor Information" :)
```suggestion
Vendor information section
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:86]: Admonition indentation
```suggestion
     The vendor list is pulled from the *Contacts* application. If a selection is made to the
     :guilabel:`Vendor` field with a drop-down menu, the :guilabel:`Vendor` field above updates
     with the name entered on the vendor form of the selected vendor.
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:104]: Optional, add the UI text in a guilabel to be specific 
```suggestion
The :guilabel:`AVAILABILITY` section presents a table with two rows. The days of the week populate
the top row, and the bottom row has checkboxes. Check the corresponding box for each day of the week
the vendor is available.
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:152]:
```suggestion
Configure extras
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:165]:
```suggestion
Add extras
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:181]: Use a period here since images shouldnt be used as part of an instruction in place of text
```suggestion
   The various toppings are then added, with their corresponding costs.
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:189]:
```suggestion
   One`. The various beverage choices are added, and the cost for each remains zero.
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:243]: No "Image" UI text so no guilabel needed :)
```suggestion
- Image: hover over the image box in the top-right, and click the :guilabel:`✏️ (pencil)` edit icon
  that appears. A file explorer pop-up window appears. Navigate to the image, then click
  :guilabel:`Open`.
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:249]:
```suggestion
Product categories
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:258]: In my testing on 2 different Runbot 17.0, I had eight preconfigured product categories?
```suggestion
Categories`. The available categories appear in a list view. In the *Lunch* app, there are eight
default categories : :guilabel:`Sandwich`, :guilabel:`Pizza`, :guilabel:`Burger`,
:guilabel:`Drinks`, :guilabel:`Pasta`, :guilabel:`Sushi`, :guilabel:`Temaki`, and
:guilabel:`Chirashi`.
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:313]: No need for a TOCTREE since there are no sub-docs
```suggestion
```

File: content/applications/hr/lunch.rst
Comment (samueljlieber) [inline content/applications/hr/lunch.rst:3]: Can remove meta tags since this is not a TOC file
```suggestion
```

File: content/applications/hr/lunch/vendor.png
Comment (samueljlieber) [inline content/applications/hr/lunch/vendor.png:1]: This image is unused and can be removed :)

**Review by samueljlieber — COMMENTED** [review]
Hi @larm-odoo! Great job on this brand new Lunch doc, I got hungry while reading through 😄 I have some suggestions for you after this peer review, please see below.

Small update to fix a typo in the commit message (use `git commit --amend`):
```
[ADD] Lunch: new configuratoin doc
⬇️
[ADD] Lunch: new configuration doc
```
Feel free to pass this over to final content review, and after that please tag me one more time for a quick technical pass. Thank you!

File: content/applications/hr/lunch.rst
Comment (larm-odoo) [reply content/applications/hr/lunch.rst:249]: I thought that since this was the text formatting in the UI menu drop-down, where you navigate to this section, it could be in title case as it appears. But I changed it- just wanted to explain why I did it, it wasn't just me falling asleep at the wheel =D

File: content/applications/hr/lunch.rst
Comment (larm-odoo) [reply content/applications/hr/lunch.rst:313]: I put that in there as a placeholder because I am adding the other docs now, lol. But I'll remove for now!

**Comment by larm-odoo** [comment]
Thanks @samueljlieber for volunteering and reviewing my doc! I noticed my typo in the commit but it was too late, and I didn't want to mess it up =D

**Comment by larm-odoo** [comment]
Hi @ksc-odoo  - this is ready for a review from you!

File: content/applications/hr/lunch.rst
Comment (StraubCreative) [inline content/applications/hr/lunch.rst:158]: Curious: is the **Only One** selection mandatory like **One or More**?

File: content/applications/hr/lunch.rst
Comment (StraubCreative) [inline content/applications/hr/lunch.rst:172]: Wording seemed off as there doesn't appear to be a clear action verb here. 
Here's a quick suggestion:
```suggestion
   For a pizzeria that only offers personal pies, see their extras configured as follows:
```

File: content/applications/hr/lunch.rst
Comment (StraubCreative) [inline content/applications/hr/lunch.rst:182]: Nitpick: transition + active tense
```suggestion
   The pizzeria also offers a free beverage with any purchase. To set this up, the :guilabel:`Extra
```

File: content/applications/hr/lunch.rst
Comment (StraubCreative) [inline content/applications/hr/lunch.rst:214]:
```suggestion
as no products are pre-configured in Odoo, by default.
```

File: content/applications/hr/lunch.rst
Comment (StraubCreative) [inline content/applications/hr/lunch.rst:222]: Another way to do this:
```suggestion
- :guilabel:`Product Name` (*required*): enter the name for the product.
```
or this way, underneath the list
```suggestion
- :guilabel:`Product Name`*: enter the name for the product.
...

(*) indicates required field
```

File: content/applications/hr/lunch.rst
Comment (StraubCreative) [inline content/applications/hr/lunch.rst:231]: This actually doesn't get a hyphen. [Source](https://en.wikipedia.org/wiki/Popover_(GUI))
```suggestion
- :guilabel:`New Until`: using the calendar popover, select the date that the product will no
```

File: content/applications/hr/lunch.rst
Comment (StraubCreative) [inline content/applications/hr/lunch.rst:236]: I *think* location queues like "top-right" are compound adjectives and they require a noun to follow them in technical writing.
```suggestion
- Image: hover over the image box in the top-right corner of the form, and click the :guilabel:`✏️ (pencil)` edit icon
```

File: content/applications/hr/lunch.rst
Comment (StraubCreative) [inline content/applications/hr/lunch.rst:282]: Can indent this, I think, to make it a child of the **Display** item above it

**Review by StraubCreative — APPROVED** [review]
Hi @larm-odoo 
Nice work, this was a pleasure to read 🙂 
Approving with comments— please minimally address grammar/wording suggestions, otherwise I think we're good to merge whenever you're ready.
.....
@robodoo delegate=larm-odoo

**Comment by StraubCreative** [comment]
@larm-odoo actually hold off on merge until we get the **Lunch Doc Review** team all set up. 
I'll coordinate with ANV and circle back.

File: content/applications/hr/lunch.rst
Comment (larm-odoo) [reply content/applications/hr/lunch.rst:222]: I love #2 - I was dividing sections into 'requied' and 'optional' but that was confusing, and not a great way to do it. I love htis versoin- much cleaner. Plus, i do like telling people what they need to enter.

**Review by samueljlieber — APPROVED** [review]

**Comment by larm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
